tests/components/zha/test_config_flow.py

```diff
@@ -134,60 +146,55 @@ async def test_probe_radios(hass):
 
         res = await config_flow.detect_radios("/dev/null")
         assert res is None
+        assert xbee_probe.await_count == 1
+        assert zigate_probe.await_count == 1
+        assert deconz_probe.await_count == 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)